product_configurator/models/product_attribute.py

```diff
@@ -45,8 +45,24 @@ def onchange_custom_type(self):
         'disable a attribute without deleting it'
     )
 
-    min_val = fields.Integer(string="Min Value", help="Minimum value allowed")
-    max_val = fields.Integer(string="Max Value", help="Minimum value allowed")
+    min_val = fields.Integer(
+        string="Min Value",
+        help="Minimum value allowed"
+    )
+    max_val = fields.Integer(
+        string="Max Value",
+        help="Maximum value allowed, or 0 if not checked"
+    )
+    min_fval = fields.Float(
+        string="Min Float Value",
+        digits=(16, 4),
+        help="Minimum value allowed"
+    )
+    max_fval = fields.Float(
+        string="Max Float Value",
+        digits=(16, 4),
+        help="Maximum value allowed, or 0 if not checked"
+    )
```

Comment:
@PCatinean True, but it needs to be a float min or max, not an integer.  Our requirement was that the minimum was 0.573, and the maximum was 20.127 (a length).

I think this is a little cleaner, though, because it implies the validation steps are dependent on the type, which could be extended to other types at another time...

Also, if a float min is used, then we need to ensure the error converts the min to an integer before displaying so it does not say "Value cannot be greater than 20.0"